README.md

```diff
@@ -1,85 +1,35 @@
-    - WebView displays error when connectivity lost
-    - AT caching logic change for scope intersection
-
-***09/30/2019*** MSAL Android is now generally available with [MSAL 1.0](https://github.com/AzureAD/microsoft-authentication-library-for-android/releases/tag/v1.0.0)!: 
-
-- Supported Authorities
-  - Microsoft [identity platform](https://docs.microsoft.com/azure/active-directory/develop/) (also known as the Azure Active Directory v2 Endpoint)
-  - [Azure Active Directory B2C](https://docs.microsoft.com/azure/active-directory-b2c/)
-    - [Integrate with B2C](https://docs.microsoft.com/azure/active-directory/develop/msal-android-b2c)
-- Microsoft [authentication broker](https://docs.microsoft.com/azure/active-directory/develop/brokered-auth) support
-  - Supports enterprise scenarios including:
-    - Device Registration
-    - Device Management
-    - Intune App Protection
-    - Device Single Sign On
-- Introduction of [Single and Multi Account](https://docs.microsoft.com/azure/active-directory/develop/single-multi-account) Public Client Applications
-- IAccount and access to claims. For more info see [Accounts and tenant profiles](https://docs.microsoft.com/azure/active-directory/develop/accounts-overview)
-- Enable Single Sign-On with different [authorization agents](https://docs.microsoft.com/azure/active-directory/develop/authorization-agents)
-- Support for synchronous methods from worker threads
-- Improved [configuration](https://docs.microsoft.com/azure/active-directory/develop/msal-configuration) and control of your PublicClientApplication using configuration file
-- AndroidX Compatible
-
-### Migrating from ADAL
-
-See the [ADAL to MSAL migration guide for Android](https://docs.microsoft.com/azure/active-directory/develop/migrate-android-adal-msal)
+[![Version Badge](https://img.shields.io/maven-central/v/com.microsoft.identity.client/msal.svg)](https://repo1.maven.org/maven2/com/microsoft/identity/client/msal/)
 
-### Sample
+### :exclamation: Migrating from ADAL
 
-Run the [quickstart](https://docs.microsoft.com/azure/active-directory/develop/quickstart-v2-android) to see how our Java sample works, or checkout this [list of all MSAL sample repos](https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki/MSAL-Code-Samples). 
+ADAL Android was deprecated on June 20223. We do not support ADAL. See the [ADAL to MSAL migration guide for Android](https://docs.microsoft.com/azure/active-directory/develop/migrate-android-adal-msal)
 
 ## Using MSAL
 
 - Before you can get a token from Azure AD v2.0 or Azure AD B2C, you'll need to register an application. To register your app, use [the Azure portal](https://aka.ms/AppRegistrations). For Azure AD B2C, checkout [how to register your app with B2C](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-app-registration).  
 
 ### Requirements
 
-- Android API Level 16+
+- Android API Level 33+
```

Comment:
I think the minSdkVersion is still set to 16, but our target sdk version recently updated to 33: https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/gradle/versions.gradle